auslib/db.py

```diff
@@ -185,7 +185,7 @@ class AUSTable(object):
     """
 
     def __init__(self, db, dialect, history=True, versioned=True, scheduled_changes=False,
-                 onInsert=None, onUpdate=None, onDelete=None):
+                 scheduled_changes_kwargs={}, onInsert=None, onUpdate=None, onDelete=None):
```

Comment:
Huh. I know there's some risk involved in using mutable values because they may end up being shared, but I didn't realize they could create global variables...